pkg/unvanquished_src.dpkdir/ui/options_graphics.rml

```diff
@@ -99,6 +99,11 @@
 					</select>
 					<h3><translate>Lighting system</translate></h3>
 				</row>
+				<row>
+					<h3><translate>Brightness clamping</translate></h3>
+					<input cvar="r_overbrightDefaultClamp" type="checkbox" />
+					<p><translate>Limit max brightness of map lighting of world surfaces and models (better compat with majority of legacy maps)</translate></p>
```

Comment:
It's better to not use abbreviations to not make the translation harder to do. Anyway I recommend to use instead something like “`Old brightness limit of map lighting of world surfaces and models (the way legacy maps looked before)`” as it is not really a matter of data compatibility but more a matter of human habituation.

Reply:
Something like "LDR lighting" might be more concise.